seed/views/v3/users.py

```diff
@@ -0,0 +1,887 @@
+                type: string
+                description: success or error
+                required: true
+            show_shared_buildings:
+                type: string
+                description: the user show shared buildings attribute
+                required: true
+            message:
+                type: string
+                description: error message, if any
+                required: false
+        """
+        ok, content = self.validate_request_user(pk, request)
+        if ok:
+            user = content
+        else:
+            return content
+
+        return JsonResponse({
+            'status': 'success',
+            'show_shared_buildings': user.show_shared_buildings,
+        })
+
+    @ajax_request_class
+    @action(detail=True, methods=['PUT'])
+    def default_organization(self, request, pk=None):
+        """
+        Sets the user's default organization
+        ---
+        parameter_strategy: replace
+        parameters:
+            - name: pk
+              description: User ID (primary key)
+              type: integer
+              required: true
+              paramType: path
+            - name: organization_id
+              description: The new default organization ID to use for this user
+              type: integer
+              required: true
+        type:
+            status:
+                type: string
+                description: success or error
+                required: true
+            message:
+                type: string
+                description: error message, if any
+                required: false
+        """
+        ok, content = self.validate_request_user(pk, request)
+        if ok:
+            user = content
+        else:
+            return content
+        user.default_organization_id = request.query_params.get('organization_id', None)
+        user.save()
+        return {'status': 'success'}
+
+    @ajax_request_class
+    @action(detail=True, methods=['DELETE'])
+    def deactivate(self, request, pk=None):
+        """
+        Deactivates a user
+        ---
+        parameter_strategy: replace
+        parameters:
+            - name:
+              description: User ID (primary key)
+              type: integer
+              required: true
+              paramType: path
+            - name: organization_id
+              description: The new default organization ID to use for this user
+              type: integer
+              required: true
+        type:
+            status:
+                type: string
+                description: success or error
+                required: true
+            message:
+                type: string
+                description: error message, if any
+                required: false
+        """
+        body = request.data
+        first_name = body['first_name']
+        last_name = body['last_name']
+
+        print(first_name, last_name)
+        # check if user exists
+        user = SEEDUser.objects.filter(
```

Comment:
I would've expected we want to use the id of the user vs `first_name` and `last_name`. Any reason to take this approach?